setup.py

```diff
@@ -4,11 +4,12 @@
 
 Run tests in transactions using pytest, Flask, and SQLAlchemy.
 '''
-import os
 from setuptools import setup
 
-with open(os.path.join('requirements', 'main.txt')) as reqfile:
-    requirements = reqfile.read().splitlines()
```

Comment:
requirements.txt are great for fixing dependencies that you know you will work for an app, but are not the best choice for a library for two reasons

1. you actually want to support the widest range of dependencies possible (counter to what you want to do with an app)
2. keeping your requirements in setup.py reduces some complexity. You could keep this approach but you would need to add 'main.txt' to the MANIFEST.in file.